client/src/featureform/register.py

```diff
@@ -1722,13 +1712,13 @@ def get_postgres(self, name):
         Returns:
             postgres (OfflineSQLProvider): Provider
         """
-        get = ProviderReference(name=name, provider_type="postgres", obj=None)
-        self.__resources.append(get)
-        fakeConfig = PostgresConfig(host="", port="", database="", user="", password="")
-        fakeProvider = Provider(
-            name=name, function="OFFLINE", description="", team="", config=fakeConfig
+        mock_config = PostgresConfig(
+            host="", port="", database="", user="", password=""
+        )
+        mock_provider = Provider(
+            name=name, function="OFFLINE", description="", team="", config=mock_config
         )
-        return OfflineSQLProvider(self, fakeProvider)
+        return OfflineSQLProvider(self, mock_provider)
 
     def get_snowflake(self, name):
```

Comment:
I'm actually really confused what these functions do again.

Reply:
prep work for now, they just return resources, eventually we'd like them to return the data as well.